packages/ui/components/elements/IngredientSearchInput/IngredientSearchInput.tsx

```diff
@@ -1,27 +1,50 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { Styled } from './IngredientSearchInput.styled'
 import { search } from 'react-icons-kit/fa/search'
 import { Icon } from 'react-icons-kit'
 import { uuidv4 } from '../../utils/uuid'
 
-type IngredientSearchInputProps = {
+interface IngredientSearchInputProps {
   id: string
   imgSrc?: string
   imgAlt?: string
+  searchIngredient: (searchTerm: string) => void
 }
 
 const SearchIcon = () => (
   <div className=''>
     <Icon icon={search} size={24} />
   </div>
 )
-const IngredientSearchInput = ({ id, imgSrc, imgAlt }: IngredientSearchInputProps) => {
+const IngredientSearchInput = ({ id, imgSrc, imgAlt, searchIngredient }: IngredientSearchInputProps) => {
   // const id = `input-search-field-${uuidv4()}`
+  const [searchTerm, setSearchTerm] = useState<string>('')
+
+  function handleChange(event: React.ChangeEvent<HTMLInputElement>) {
+    let value = event?.target?.value
+    if (!value) value = ''
+    setSearchTerm(value)
```

Comment:
can we use this ?
function handleChange..........
  const value = event?.target?.value || '';
  setSearchTerm(value);
}

Reply:
Yes, we can. Thanks.